k8s/voucher/apptest/tester/tester.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+curl -X POST ${APP_INSTANCE_NAME}-service/snakeoil
```

Comment:
Does it check anything?  Will curl exit with an error code if it gets a 4xx?

Reply:
It just checks the voucher server is up and running (it's hard to check much beyond that). and yeah, it exits with error if the server is unreachable etc.